src/native-common/Image.tsx

```diff
@@ -46,6 +46,10 @@ export class Image extends RX.Image<{}> {
     private _nativeImageWidth: number;
     private _nativeImageHeight: number;
 
+    protected _getAdditionalProps(): { [key: string]: string|number } {
```

Comment:
Should we make the return value of type `ImageProps` with everything optional? I think that would be expressed as `Partial<ImageProps>`.